### DISCUSSION TITLE
What the Future Holds for eCash (XEC)

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6220] [General]
https://swapzone.io/blog/what-the-future-holds-for-ecash

![BLOG-4-1024x538](https://github.com/horizontalsystems/unstoppable-wallet-android/assets/113377431/cc7b3ee4-aca0-4075-b586-4b14a0f38144)

**Comment by 3point14159** [comment]
Making things happen! Love the namespace concept, can't wait to get mine. =)
